site.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
+/* global getPhoto, getTopCountriesList, milesBetweenPoints, linkPoints, getCountryClues */
+
+var TOTAL_ROUNDS = 5;
+var roundNumber = 1;
```

Comment:
Does this need to be set globally in the code as is? It's being set also within the main.